docs/runtime/modules.md

```diff
@@ -263,7 +263,9 @@ await Bun.build({
 
 ## Path re-mapping
 
-In the spirit of treating TypeScript as a first-class citizen, the Bun runtime will re-map import paths according to the [`compilerOptions.paths`](https://www.typescriptlang.org/tsconfig#paths) field in `tsconfig.json`. This is a major divergence from Node.js, which doesn't support any form of import path re-mapping.
+In the spirit of treating TypeScript as a first-class citizen, the Bun runtime will re-map import paths according to the [`compilerOptions.paths`](https://www.typescriptlang.org/tsconfig#paths) field in `tsconfig.json`.
+
+This behavior differs from Node.js, which only supports import path re-mapping via the `"imports"` field in `package.json`, and requires that mapped paths [always start with `#`](https://nodejs.org/api/packages.html#subpath-imports).
```

Comment:
Instead of highlighting how it's different in Bun, we should highlight how there are two options - using editor-friendly tsconfig.json "paths", or using Node.js-friendly "#imports"